packages/calypso-products/src/get-domain.js

```diff
@@ -1,19 +1,12 @@
-/**
- * External dependencies
- */
-import { get } from 'lodash';
-
 /**
  * Internal dependencies
  */
-import { assertValidProduct } from './utils/assert-valid-product';
 import { formatProduct } from './format-product';
 
 export function getDomain( product ) {
 	product = formatProduct( product );
-	assertValidProduct( product );
 
-	const domainToBundle = get( product, 'extra.domain_to_bundle', false );
+	const domainToBundle = product.extra?.domain_to_bundle;
```

Comment:
I wonder if we can shorten this to:

```suggestion
	const domainToBundle = product.extra?.domain_to_bundle ?? product.meta;
```

and that way remove these last lines from the function.

Reply:
👍